README.md

```diff
@@ -65,13 +66,13 @@ To use the library, add the following dependencies to your `platformio.ini`'s `l
 
 ```control
 lib_deps =
-    Sensirion UPT I2C Auto Detection
+    Sensirion/Sensirion UPT I2C Auto Detection
```

Comment:
does it work with the first sensirion capitalized ?
see https://registry.platformio.org/libraries/sensirion/Sensirion%20UPT%20Core

Reply:
Cannot test since this lib was not yet released. However
```pio pkg install --library "Sensirion/Sensirion UPT Core"```
works as advertised, so I expect it to work. But perhaps it's a good idea to not capitalize

Reply:
In fact it looks like it is not case-sensitive